src/code/z_vr_box_draw.c

```diff
@@ -20,7 +20,7 @@ void SkyboxDraw_Draw(SkyboxContext* skyboxCtx, GraphicsContext* gfxCtx, s16 skyb
     gSPSegment(POLY_OPA_DISP++, 0x8, skyboxCtx->staticSegments[1]);
     gSPSegment(POLY_OPA_DISP++, 0x9, skyboxCtx->staticSegments[2]);
 
-    gDPSetPrimColor(POLY_OPA_DISP++, 0x00, 0x00, 0, 0, 0, alpha);
+    gDPSetPrimColor(POLY_OPA_DISP++, 0x00, 0x00, 0, 0, 0, blend);
```

Comment:
why "blend" ?

Reply:
its not really the alpha value of any one skybox, its used to blend between two different ones. `Kankyo_UpdateSkybox` makes that swapping and blending a bit more clear